io.github.rocket.editor.appdata.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright 2023 Erik Faye-Lund <kusmabite@gmail.com> -->
+<component type="desktop-application">
+  <id>io.github.rocket.editor</id>
+  <metadata_license>FSFAP</metadata_license>
+  <project_license>Zlib</project_license>
+  <name>Rocket</name>
+  <summary>Rocket Editor</summary>
+
+  <description>
+    <p>
+      Rocket is an intuitive new way of... bah, whatever. It's a
+      sync-tracker, a tool for synchronizing music and visuals in
+      demoscene productions. It consists of a GUI editor (using Qt5), and
+      an ANSI C library that can either communicate with the editor over
+      a network socket, or play back an exported data-set.
+    </p>
+  </description>
+
+  <launchable type="desktop-id">io.github.rocket.editor.desktop</launchable>
+
+  <screenshots>
+    <screenshot type="default">
+      <image>https://user-images.githubusercontent.com/47073/220860003-4feef034-a937-4fdc-9e27-7cb4d710516c.png</image>
+    </screenshot>
+  </screenshots>
+
+  <url type="homepage">https://rocket.github.io/</url>
+
+  <content_rating type="oars-1.1" />
+
+  <releases>
+    <release version="0.11.0" date="2018-08-01" />
+  </releases>
```

Comment:
It's a bit unclear to me if the releases should be *all* releases ever made (including source-only releases), or just the ones for this flatpak. If it's the former, I guess I'm missing a few here. If it's the latter, the question becomes if the date should be the date of the source-release or the flatpak release. Those differ quite widely in this case...

Reply:
It only requires the latest release. You can add former if you want, so Software Centers can show a full changelog, but it is not required.

Reply:
it's always when upstream released. I usually take the tag date if I have to guess, or the tarball date.